include/tester/ssp/Softphone.hpp

```diff
@@ -12,17 +12,16 @@ class Softphone
 {
 public:
     Softphone(const SoftphoneArguments & args):
-        _account(args.id, args.domain, args.secret, std::bind(&inComingCall, *this, std::placeholders::_1)),
-        _call(&_account, std::bind(&onCallState, *this, std::placeholders::_2)) //TODO
+        _account(args.id, args.domain, args.secret, std::bind(&Softphone::onInComingCall, this, std::placeholders::_1))
     {
+        _call = new SSPCall(&_account, boundOnCallState);
         _account.applyAccount();
     }
 
     ~Softphone();
 
-    void call(std::string & id_to_call, std::string & domain)
-    {  
-        _call = new SSPCall(*this); //TODO
+    void call(const std::string & id_to_call, const std::string & domain)
```

Comment:
make this method take a sip URI instead of these two parameters.

Reply:
done